torchvision/datasets/video_utils.py

```diff
@@ -55,17 +61,17 @@ class _VideoTimestampsDataset(object):
     pickled when forking.
     """
 
-    def __init__(self, video_paths: List[str]):
+    def __init__(self, video_paths: List[str]) -> None:
         self.video_paths = video_paths
 
-    def __len__(self):
+    def __len__(self) -> int:
         return len(self.video_paths)
 
-    def __getitem__(self, idx):
+    def __getitem__(self, idx: int) -> Tuple[List[int], Optional[float]]:
         return read_video_timestamps(self.video_paths[idx])
 
 
-def _collate_fn(x):
+def _collate_fn(x: Any) -> Any:
```

Comment:
Can we make this more concrete?

```suggestion
from typing import TypeVar

T = TypeVar("T")

def _collate_fn(x: T) -> T:
```

Without that, the output of `collate_fn` will always be `Any` and thus effectively loosing us the information we have about the input type.